gensim/scripts/glove2word2vec.py

```diff
@@ -5,18 +5,36 @@
 # Copyright (C) 2016 Manas Ranjan Kar <manasrkar91@gmail.com>
```

Comment:
Look at https://pythonhosted.org/sphinxcontrib-programoutput/, this should be helpful for documenting CLI applications :+1: 

Add it to current sphinx-build and test, how it works